Misc/NEWS.d/next/Library/2019-08-11-16-28-03.bpo-26543.X-TJZO.rst

```diff
@@ -0,0 +1 @@
+Fix crash on :meth:`IMAP4.noop()` when bytes data is passed.
```

Comment:
For me, a "crash" means that the Python process is killed immediately, like a SIGSEGV. Here it's "just" a regular exception. Moreover, you should mention that the issue only affects debug mode. I suggest:

```
Fix :meth:`IMAP4.noop()` when debug mode is enabled (ex: ``imaplib.Debug = 3``).
```